tasks/upgrade.yaml

```diff
@@ -32,6 +32,10 @@ tasks:
         description: The architecture of the package to create
         default: ${UDS_ARCH}
 
+      dep_commands:
+        description: Additional commands to run inside the old repo to setup dependencies
```

Comment:
I may be missing context, but what is meant "run inside the old repo..."

Reply:
The upgrade task here clones in the older repo at the last tag to make the bundle to handle version updates more dynamically (useful if chart names are added or packages change in more significant ways that are incompatible) - that's what all the git stuff is doing toward the top of the task.